python/lsst/daf/butler/script/transferDatasets.py

```diff
@@ -79,12 +78,12 @@ def transferDatasets(
     dest_butler = Butler.from_config(dest, writeable=True)
 
     dataset_type_expr = dataset_type or ...
-    collections_expr: tuple[str, ...] | EllipsisType = collections or ...
+    collections_expr: tuple[str, ...] = collections or tuple("*")
```

Comment:
```suggestion
    collections_expr: tuple[str, ...] = collections or ("*",)
```
It amounts to the same thing here, because it's a single character, but this makes it look less like a mistake.